typing/printtyp.ml

```diff
@@ -1814,11 +1814,9 @@ let explain_escape intro ctx e =
     | None -> ignore
     | Some ctx ->  dprintf "@[%t@;<1 2>%a@]" intro type_expr ctx in
   match e with
-  | Trace.Univ Some u ->  Some(
-      dprintf "%t@,The universal variable '%s would escape its scope"
-        pre u)
-  | Trace.Univ None ->
-      Some(dprintf "%t@,An universal variable would escape its scope" pre)
```

Comment:
What does the `None` case become in the new code?

Reply:
A free type variable name (`'a`, `'b`,  ..., `'a1`, ... )  is picked for the unnamed universal variable like it happens in the unification error for `let f: _ -> _ = ()`.

Reply:
Ok, sounds good then.